packages/constants/src/index.ts

```diff
@@ -72,6 +72,7 @@ export * from './helpers'
 export * from './mcp'
 export * from './evaluations'
 export * from './integrations'
+export * from './latte'
```

Comment:
Don't add to the index barrel. Put a new export line in the package.json

Reply:
Is bad for code splitting. Let's try not to use barrel files

Reply:
+1